### PR TITLE
Term entry c sharp sin

### LINKED ISSUE #1614: [Term Entry] C# Math Functions `.Sin()`
### Reason/inspiration (optional)

We would like a new term entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the `Math.Sin()` method.

The entry should go in a new file in `docs/content/c-sharp/concepts/math-functions/terms/sin/sin.md`

The entry should

- Have a syntax section illustrating the method with a ` ```pseudo ` code block.
- An example showing the method in use with a ` ```cs ` code block.
- A runnable codebyte example in a ` ```codebyte/csharp ` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new term for C# Math Functions: Math.Sin()
- Closes #1614 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/sin/sin.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The **`Math.Sin()`** class method returns the sine of a given angle.
+
+## Syntax
+
+```pseudo
+Math.Sin(angle);
+```
+
+The `Math.Sin()` method takes only one `double` type parameter - `angle` - an angle in radians. The `Math.Sin()` method returns the sine of the `angle` as a `double` value  or `NaN` (not a number) if the `angle` equals to:
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sin/sin.md:25]:
```suggestion
The `Math.Sin()` method takes only one parameter,`angle`, an angle in radians of type `double`. The method returns the sine of the `angle` as a `double` value  or `NaN` (not a number) if the value of `angle` equals:
```

File: content/c-sharp/concepts/math-functions/terms/sin/sin.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+- `NaN`,
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sin/sin.md:27]:
```suggestion
- `NaN`
```

File: content/c-sharp/concepts/math-functions/terms/sin/sin.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- `NegativeInfinity`, or
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sin/sin.md:28]:
```suggestion
- `NegativeInfinity`
```

File: content/c-sharp/concepts/math-functions/terms/sin/sin.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+- `PositiveInfinity`.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sin/sin.md:29]:
```suggestion
- `PositiveInfinity`
```

File: content/c-sharp/concepts/math-functions/terms/sin/sin.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+    Console.WriteLine("The sine of " + degrees + " degrees is: " + sine);
+  }
+}
+```
+
+The example will result in the following output:
+
+```shell
+The sine of 30 degrees is: 0.5
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sin/sin.md:53]: I'm getting .49999... Perhaps there should be a rounding method included.

File: content/c-sharp/concepts/math-functions/terms/sin/sin.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+The following example is runnable and returns the sine of the `angle` given in degrees:
+
+```codebyte/csharp
+using System;
+
+public class Example {
+
+  public static void Main(string[] args) {
+    //angle in degrees
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sin/sin.md:66]:
```suggestion
    // Angle in degrees
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla I've completed my initial review. Please update and tag me when you've finished making changes. Thanks

File: content/c-sharp/concepts/math-functions/terms/sin/sin.md (hunk 5 above)
Comment (cslylla) [reply content/c-sharp/concepts/math-functions/terms/sin/sin.md:53]: I get 0.5. It should be 0.5. 
sine
I am trying to figure out what the issue is. Even if I use Math.Round(), I will get 0.5 and not 0.49999. Do you know what the issue could be?

**Comment by cslylla** [comment]
@caupolicandiaz I accepted the changes, but I am not sure what the issue is with the example.

File: content/c-sharp/concepts/math-functions/terms/sin/sin.md (hunk 5 above)
Comment (cslylla) [reply content/c-sharp/concepts/math-functions/terms/sin/sin.md:55]: I added a note that explains the difference in the results.

File: content/c-sharp/concepts/math-functions/terms/sin/sin.md [hunk 7]
```diff
@@ -22,15 +22,17 @@ The **`Math.Sin()`** class method returns the sine of a given angle.
...
+The `Math.Sin()` method takes only one parameter, `angle`, an angle in radians of type `double`. The method returns the sine of the `angle` as a `double` value  or `NaN` (not a number) if the value of `angle` equals:
 
-- `NaN`,
-- `NegativeInfinity`, or
-- `PositiveInfinity`.
+- `NaN`
+- `NegativeInfinity`
+- `PositiveInfinity`
+
+> **Note:** Depending on the used operating system or architecture, the exact result or the input range of the `Math.Sin()` method may differ due to the differences in the C runtime environment.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sin/sin.md:31]: As I mentioned in the other entry I think we can move this note down and edit out the rounding step from the examples. Thanks

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla A couple of edits.

**Comment by cslylla** [comment]
@caupolicandiaz I moved the note and removed the rounding from the examples.

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz I moved the note and removed the rounding from the examples.

This looks good. I ultimately decided not to include the note since I'm not confident in what's actually at the root of the discrepancy. This is ready for another review.

**Review by caupolicandiaz — APPROVED** [review]

**Review by KTom101 — APPROVED** [review]
@cslylla This looks good to me as is.

**Comment by KTom101** [comment]
@cslylla Entry is live: https://www.codecademy.com/resources/docs/c-sharp/math-functions/sin 🎉

Screenshot 2023-03-23 at 9 38 40 PM
 
cc @yangc95 @SSwiniarski

**Pull request merged**